src/Symfony/Component/Form/DataMapperInterface.php

```diff
@@ -52,9 +54,11 @@ public function mapDataToForms(mixed $viewData, \Traversable $forms);
      * The model data can be an array or an object, so this second argument is always passed
      * by reference.
      *
-     * @param FormInterface[]|\Traversable $forms    A list of {@link FormInterface} instances
-     * @param mixed                        $viewData The compound form's view data that get mapped
-     *                                               its children model data
+     * @param \Traversable<mixed, FormInterface> $forms    A list of {@link FormInterface} instances
```

Comment:
do we really have to use mixed for the key? can't we omit it?

Reply:
All static analyzers use stubs with two template variables afaik. And `mixed` is the closest thing to "I don't care" I could come up with. 🤷

Reply:
Omitting the key template variable seems to work though:
* https://phpstan.org/r/ed0300e9-008a-4bfc-84b4-3d54c7be8bfb
* https://psalm.dev/r/524d8b2350

TIL 🤓